### PR TITLE
fix: fix logic bug and compile errors on native ads

### PR DESCRIPTION
**Pull request by BernieSumption**
Minor bugfix to get ads stories working on native

### CODE REVIEW

File: packages/ad/ad.stories.js [hunk 1]
```diff
@@ -176,7 +175,7 @@ storiesOf("Primitives/Advertisement", module)
                   width: 100%;
                   height: 100%;
                   background: ${worked ? "#8C8" : "#C88"};
-                  font-size: ${fontSizes.smallestHeadline}px;
+                  font-size: 20px;
                   padding: 10px;
```
Comment (nikhedonia) [inline packages/ad/ad.stories.js]: please use styleguide.spacing

File: packages/ad/ad.stories.js [hunk 2]
```diff
@@ -176,7 +175,7 @@ storiesOf("Primitives/Advertisement", module)
                   width: 100%;
                   height: 100%;
                   background: ${worked ? "#8C8" : "#C88"};
```
Comment (nikhedonia) [inline packages/ad/ad.stories.js]: do we have a matching color in our styleguide ? 
if not, please us 6 char color codes (we agreed on that once and it makes it easier to do find/replace when refactoring)

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e6155d0c370f516b136bbe86d3d1f5a813d6e891/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e247cef7a18241a325e0e8eb7779ff622de2953d/index.html

**Pull request merged**